website/docs/tasks/simulate.md

```diff
@@ -2,61 +2,5 @@
 title: simulate
 ---
 
-## Overview
-
-Taq `simulate` is used to simulate the behavior of an entrypoint in a given smart contract. It takes an initial storage value, and the input parameters of the entrypoint and will execute the associated code on a running Flextesa sandbox, and return the results
```

Comment:
Let's make a (temporary) note here and mention the changes (moving away from the coupling to Flextesa for example)

Reply:
Will do!